R/params.R

```diff
@@ -0,0 +1,66 @@
+#' Generate a dataframe of default parameters for a benchmark
+#'
+#' Generates a dataframe of parameter combinations for a benchmark to try based
+#' on the parameter defaults of its `setup` function and supplied parameters.
+#'
+#' @param x An object for which to generate parameters
+#' @param ... Named arguments corresponding to the parameters of `bm`'s `setup`
+#' function. May also contain `cpu_count`, `lib_path`, and `mem_alloc`.
+#'
+#' @return For `get_default_parameters.Benchmark`, a dataframe of parameter combinations
+#' to try with a column for each parameter and a row for each combination.
+#'
+#' @export
+get_default_parameters <- function(x, ...) {
+  UseMethod("get_default_parameters")
+}
+
+#' @export
```

Comment:
Do we want or need `#' @rdname get_default_parameters` here?

Reply:
Nothing complains if you don't, but it does make it easier to see what classes have methods, so I'll add it